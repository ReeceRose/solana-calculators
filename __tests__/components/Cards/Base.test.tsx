import { render } from '@testing-library/react';

import BaseCard from '../../../components/Cards/Base';

test('Base card component matches snapshot', () => {
  const { asFragment } = render(<BaseCard> </BaseCard>);
  expect(asFragment()).toMatchSnapshot();
});
