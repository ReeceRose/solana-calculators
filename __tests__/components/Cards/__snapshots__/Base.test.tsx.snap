// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Base card component matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="relative flex flex-col w-full min-w-0 mb-12 break-words border-0 rounded-lg shadow-lg bg-gray-700"
  >
    <div
      class="px-6 py-6 mb-0 bg-white rounded-t bg-gray-900"
    >
      <div
        class="flex justify-between text-center"
      >
        <h6
          class="text-xl font-bold text-solana-green"
        >
          Solana Utilities
        </h6>
      </div>
    </div>
     
  </div>
</DocumentFragment>
`;
